common/connections.py

```diff
@@ -5,79 +5,87 @@
 from re import search
 from typing import Dict, Callable, Any
 
+
 class Client:
-    def __init__(self, baseUrl:str):
+    def __init__(self, baseUrl: str):
         self.__baseUrl = baseUrl
         self.__ticket = None
         self.__session = None
-        
-    def __updateTicket(self, ticket:str=None):
-        if ticket != None:
+
+    def __updateTicket(self, ticket: str = None):
+        if ticket is not None:
             self.__ticket = ticket
 
-        if self.__ticket != None and self.__session != None:
+        if self.__ticket is not None and self.__session is not None:
             basicStr = 'BASIC ' + b64encode(bytes(self.__ticket, 'utf-8')).decode('utf-8')
             self.__session.headers.update({'Authorization': basicStr})
-            
-    def __processRequest(self, api:str, opUrl:str, version:str, params:Dict, headers:Dict, responseHandler:Callable[[Response], Any], errorHandler:Callable[[Response], Any], requestHandler:Callable[[str,Dict,Dict,Any], Any], payload:Any=None):
+
+    def __processRequest(self, api: str, opUrl: str, version: str, params: Dict, headers: Dict, responseHandler: Callable[[Response], Any], errorHandler: Callable[[Response], Any], requestHandler: Callable[[str, Dict, Dict, Any], Any], payload: Any = None, files: Dict = None):
```

Comment:
I believe the requestHandler signature needs to be updated to handle the additional effectiveFiles parameter passed after the changes

Reply:
ok, done